### DISCUSSION TITLE
Is Flask-OpenAPI3 an alternative to Flask-restx?

### DISCUSSION THREAD

**Discussion by jpiwek** [luolingchun/flask-openapi3#162] [Q&A]
Hi,

I was using Flask2.3 with Flask-restx in the past to create REST endpoints. On top I ran a React frontend.
For future Flask3 REST Endpoint applications I need something more reliable in terms of long term support.
Would be nice if Flask-OpenAPI3 is there right way to go for.

**Comment by luolingchun** [comment]
@jpiwek 

Flask-OpenAPI3 is not a replacement for Flask-restx, and the two have different design philosophies and different ways to use them.

Flask-OpenAPI3 has been in my production environment for more than 2 years, and you are welcome to use Flask-OpenAPI3.
